src/macros-by-example.md

```diff
@@ -23,7 +23,7 @@
 > _MacroMatch_ :\
 > &nbsp;&nbsp; &nbsp;&nbsp; [_Token_]<sub>_except $ and delimiters_</sub>\
 > &nbsp;&nbsp; | _MacroMatcher_\
-> &nbsp;&nbsp; | `$` [IDENTIFIER] `:` _MacroFragSpec_\
+> &nbsp;&nbsp; | `$` ( [IDENTIFIER_OR_KEYWORD] | [RAW_IDENTIFIER] | `_` ) `:` _MacroFragSpec_\
```

Comment:
Thanks!  Can this also mention that `$crate` isn't allowed?  Maybe something like:

```suggestion
> &nbsp;&nbsp; | `$` ( [IDENTIFIER_OR_KEYWORD]<sub>_except `crate`_</sub> | [RAW_IDENTIFIER] | `_` ) `:` _MacroFragSpec_\
```